action.yml

```diff
@@ -4,30 +4,37 @@ branding:
   icon: 'cloud'
   color: 'orange'
 inputs:
-  registries:
+  http-proxy:
     description: >-
-      A comma-delimited list of AWS account IDs that are associated with the ECR Private registries.
-      If you do not specify a registry, the default ECR Private registry is assumed.
-      If 'public' is given as input to 'registry-type', this input is ignored.
+      Proxy to use for the AWS SDK agent.
     required: false
-  skip-logout:
+  mask-password:
     description: >-
-      Whether to skip explicit logout of the registries during post-job cleanup.
-      Exists for backward compatibility on self-hosted runners.
-      Not recommended.
+      Mask the docker password to prevent it being printed to logs to std-out. This will prevent the
+      password GitHub output from being shared between separate jobs.
```

Comment:
```suggestion
      Mask the docker password to prevent it from appearing in debug logs. This will prevent the
      password used to login to Amazon ECR from being shared between jobs.